src/Bitcoin/Entry.cpp

```diff
@@ -15,93 +15,92 @@ using namespace TW::Bitcoin;
 using namespace TW;
 using namespace std;
 
-bool Entry::validateAddress(TWCoinType coin, const string& address, byte p2pkh, byte p2sh, const char* hrp) const {
+bool Entry::validateAddress(TWCoinType coin, const string& address, byte p2pkh, byte p2sh,
+                            const char* hrp) const {
     switch (coin) {
-        case TWCoinTypeBitcoin:
-        case TWCoinTypeDigiByte:
-        case TWCoinTypeLitecoin:
-        case TWCoinTypeMonacoin:
-        case TWCoinTypeQtum:
-        case TWCoinTypeViacoin:
-        case TWCoinTypeBitcoinGold:
-            return SegwitAddress::isValid(address, hrp)
-                || Address::isValid(address, {{p2pkh}, {p2sh}});
-
-        case TWCoinTypeBitcoinCash:
-            return BitcoinCashAddress::isValid(address)
-                || Address::isValid(address, {{p2pkh}, {p2sh}});
-
-        case TWCoinTypeECash:
-            return ECashAddress::isValid(address)
-                   || Address::isValid(address, {{p2pkh}, {p2sh}});
-
-        case TWCoinTypeDash:
-        case TWCoinTypeDogecoin:
-        case TWCoinTypeRavencoin:
-        case TWCoinTypeFiro:
-        default:
-            return Address::isValid(address, {{p2pkh}, {p2sh}});
+    case TWCoinTypeBitcoin:
+    case TWCoinTypeDigiByte:
+    case TWCoinTypeLitecoin:
+    case TWCoinTypeMonacoin:
+    case TWCoinTypeQtum:
+    case TWCoinTypeViacoin:
+    case TWCoinTypeBitcoinGold:
+        return SegwitAddress::isValid(address, hrp) || Address::isValid(address, {{p2pkh}, {p2sh}});
+
+    case TWCoinTypeBitcoinCash:
+        return BitcoinCashAddress::isValid(address) || Address::isValid(address, {{p2pkh}, {p2sh}});
+
+    case TWCoinTypeECash:
+        return ECashAddress::isValid(address) || Address::isValid(address, {{p2pkh}, {p2sh}});
+
+    case TWCoinTypeDash:
+    case TWCoinTypeDogecoin:
+    case TWCoinTypeRavencoin:
+    case TWCoinTypeFiro:
+    default:
+        return Address::isValid(address, {{p2pkh}, {p2sh}});
     }
 }
 
 string Entry::normalizeAddress(TWCoinType coin, const string& address) const {
     switch (coin) {
-        case TWCoinTypeBitcoinCash:
-            // normalized with bitcoincash: prefix
-            if (BitcoinCashAddress::isValid(address)) {
-                return BitcoinCashAddress(address).string();
-            } else {
-                return std::string(address);
-            }
-
-        case TWCoinTypeECash:
-            // normalized with ecash: prefix
-            if (ECashAddress::isValid(address)) {
-                return ECashAddress(address).string();
-            } else {
-                return std::string(address);
-            }
-
-        default:
-            // no change
-            return address;
+    case TWCoinTypeBitcoinCash:
+        // normalized with bitcoincash: prefix
+        if (BitcoinCashAddress::isValid(address)) {
+            return BitcoinCashAddress(address).string();
+        } else {
+            return {address};
+        }
```

Comment:
```suggestion
        }
        return address;
```

`else` after `return` is not needed